pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -133,7 +209,6 @@ def list_variable(self):
         """
         return self._list_variable
 
-#==============================================================================
```

Comment:
These separators help with the readability. Why is this removed?

Reply:
Must've removed them while resolving conflicts somehow, i'll put them back

Reply:
fixed in ecb58740e3f5b11a07555ef325f3933982e58160